server/channels/web/oauth_test.go

```diff
@@ -272,7 +272,8 @@ func TestOAuthAccessToken(t *testing.T) {
 	require.NoError(t, err)
 	rurl, _ := url.Parse(redirect)
 
-	apiClient.Logout(context.Background())
+	res, err := apiClient.Logout(context.Background())
+	require.NoError(t, err)
```

Comment:
You don't need that variable
```suggestion
	_, err := apiClient.Logout(context.Background())
	require.NoError(t, err)
```